submissions/joyChukwukere/html-forms/html-forms.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Page</title>
+    </head>
+    <body>
+        <h1>Contact Us</h1>
+        <form action="#" method="POST" enctype="multipart/form-data">
+            <label for="firstname">First Name</label> <br>
+            <input type="text" name="firstname" id="firstname"/> <br> <br>
+            <label for="lastname">Last Name</label> <br>
+            <input type="text" name="lasttname" id="lastname"/> <br> <br>
+            <label for="emaIL">Email:</label> <br>
+            <input type="text" name="email" id="email" placeholder="Enter your email address."/> <br> <br>
+
+            <label>Subjects</label> <br>
+            <input list="subjects-list" name="subjects1" id="subjects"/>
+            <datalist id="subjects-list">
+                <option value="Refund">
+                <option value="Damaged product">
+                <option value="Return generation">
+                <option value="Enquires">
+                <option value="Other issues">
+            </datalist> <br> <br>
+
+            <label for="message">Leave Your Message</label>  <br>
+            <textarea name="message" id="message" cols="50" rows="10"></textarea> <br>  <br>
+
+            <button onclick="alert('Thanks for contacting us!')">Submit</button>
```

Comment:
Remove the js. This is html form task. No styling or JS should be added. follow instructions given in the task strictly

Reply:
Hi Adaa,
Thanks for the help the other day.
Please I do not understand the comment. I did only the things we were taught. By Js, do you mean <button onclick> part or the <text area>?

Reply:
This is javascript `onclick="alert('Thanks for contacting us!')` . Remove it, it is not needed

Reply:
ok
Thanks

Reply:
I have removed it and i did add, commit and push. Please review and merge. Thanks!